src/frameworks/ez.md

```diff
@@ -1,21 +1,146 @@
+By default, eZ Platform is configured to use a single Redis instance for both the application cache and session
+storage.  You may optionally choose to use a separate Redis instance for session storage in case you have a lot of
+authenticated traffic (and thus there would be many session records).
 
-To do so, uncomment the `redissession` entry in the `.platform/services.yaml` file and the corresponding relationship in the `.platform.app.yaml` file.  The bridge code that is provided with eZ Platform 1.13 and later will automatically detect the additional Redis service and use it for session storage.
+To do so, uncomment the `redissession` entry in the `.platform/services.yaml` file and the corresponding relationship
+in the `.platform.app.yaml` file.  The bridge code that is provided with eZ Platform 1.13 and later will automatically
+detect the additional Redis service and use it for session storage.
 
 ## Modifying an existing eZ Platform project
 
-If you have an existing eZ Platform project that was upgraded from a previous version, or want to resynchronize with the latest recommended configuration, please see the [eZ Platform official repository](https://github.com/ezsystems/ezplatform).
+If you have an existing eZ Platform project that was upgraded from a previous version, or want to resynchronize with
+the latest recommended configuration, please see the 
+[eZ Platform official repository](https://github.com/ezsystems/ezplatform).
 
 In particular, see:
  
- * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
+ * The [.platform.app.yaml](https://github.com/ezsystems/ezplatform/blob/master/.platform.app.yaml) file, which
+ automatically builds eZ Platform in dev mode or production mode depending on your defined project-level variables.
  * The [.platform](https://github.com/ezsystems/ezplatform/tree/master/.platform) directory
- * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically enable Redis-based cache and session support if detected.
+ * The [platformsh.php](https://github.com/ezsystems/ezplatform/blob/master/app/config/env/platformsh.php) configuration
+  file, which does the work of mapping Platform.sh environment variables into eZ Platform.  It also will automatically 
+  enable Redis-based cache and session support if detected.
+
+
+## Local Development with eZ Platform 2.x and later
+
+eZ systems provide a tool called [eZ Launchpad](https://ezsystems.github.io/launchpad/) for local development that
+improves Developer eXperience and reduces the container management. eZ Launchpad is ready to work with Platform.sh.
```

Comment:
This paragraph should discuss how it's build atop Docker for local dev, as otherwise Docker just sort of appears in the instructions further down with no explanation.

Reply:
I have changed it, let me know.